aoservicelisting.civix.php

```diff
@@ -175,34 +195,31 @@ public static function endABARelationship($values, $rowNumber, $orgId) {
 
   public static function findDupes($cid, $orgId, &$individualParams, $rel = EMPLOYER_CONTACT_REL, $isOrgOptional = FALSE) {
     if (!empty($cid)) {
-      $currentDetails = civicrm_api3('Contact', 'getsingle', ['id' => $cid]);
```

Comment:
@Edzelopez why are we removing these lines of code? if the name changes for any of the regulated staff or ABA staff (that isn't the primary contact) then shouldn't we be ending the relationship?